app/uk/gov/hmrc/soletraderidentificationfrontend/forms/CaptureFullNameForm.scala

```diff
@@ -39,12 +39,11 @@ class CaptureFullNameForm {
 
   private val firstNameInvalid: Constraint[String] = Constraint("first_name.invalid")(
     firstName => validateNot(
-      constraint = validName(firstName.capitalize.trim),
+      constraint = validName(firstName.capitalize),
       errMsg = "error.invalid_first_name"
```

Comment:
decided to remove the trim here as now trim happens before validation, so no need to do it twice